packages/ui/src/AssetIcon/group.tsx

```diff
@@ -50,16 +50,29 @@ export const AssetGroup = ({
   hideBadge,
 }: AssetGroupProps) => {
   if (variant === 'split') {
+    const firstAssetIsDelegated = assets?.[0]?.symbol === 'delUM';
+    const secondAssetIsDelegated = assets?.[1]?.symbol === 'delUM';
```

Comment:
We should add this to the metadata fields instead of relying on this ; this seems brittle. We can fix this later though.